lua/materials/cfc_trainfuck_kill_icon.vmt

```diff
@@ -0,0 +1,5 @@
+"LightmappedGeneric"
+{
+	"$basetexture" "cfc_trainfuck_kill_icon"
+	"$translucent" 1
```

Comment:
Tabs here, tho i'm not sure if you can use spaces in vmt files, might wanna look into that